src/components/modals/WalletListModal.tsx

```diff
@@ -123,22 +112,11 @@ export function WalletListModal(props: Props) {
 
   // #region Init
 
-  // Upgrade deprecated props
-  const [legacyAllowedAssets, legacyExcludeAssets] = React.useMemo(() => {
-    if (allowedCurrencyCodes == null && excludeCurrencyCodes == null) return []
-
-    const lookup = makeCurrencyCodeTable(account.currencyConfig)
-    const allowedAssets = upgradeCurrencyCodes(lookup, allowedCurrencyCodes)
```

Comment:
Since `WalletListModal` no longer uses `upgradeCurrencyCodes`, we should move this helper into its own utility file. Please create "src/utils/tokenIdTools.ts", an put `upgradeCurrencyCodes` in there, along with the `makeCurrencyCodeTable` function that goes along with it.

That will complete the cleanup for this feature.